### DISCUSSION TITLE
stream.m3u8 broken in 0.6.3?

### DISCUSSION THREAD

**Discussion by scoot-git** [mrlt8/docker-wyze-bridge#112] [Q&A]
Sorry, super newbie here.  I tried to build 0.6.0 to confirm but I have no idea what to do to accomplish this.
Can anyone confirm stream.m3u8 output does or doesn't work?  rtsp:// is fine.
I believe 0.6.0 and maybe 0.6.1 was fine and definitely ok prior to those.
It could be me though.  I'm getting blocky gibberish on my screen for display with stream.m3u8 using VLC on Windows 10 cam is V3

**Comment by mrlt8** [comment]
Thanks, I'll have a look.

Should be able to specify the tag in your docker-compose:
```yaml
image: mrlt8/wyze-bridge:0.6.1
```

There were a couple of change made to improve the robustness of the stream that could have broken something. I believe rtsp-simple-server was also updated around the same time.

**Comment by scoot-git** [comment]
0.6.1 works

**Comment by scoot-git** [comment]
Just tried 0.6.2.  that's where it got broken.

**Comment by mrlt8** [comment]
Ok, I think this should be fixed in v0.6.4. 

RTMP and HLS supposedly don't like when the resolution changes and the cameras seem to change the resolution on their own or when opening the app. 

Anyways, it should now attempt to restart the stream if the resolution doesn't match for 30 consecutive frames. You can play around with the number of frames by setting your own number in `BAD_FRAMES=30`.

**Reply by scoot-git** [reply]
Looks like it's back to AOK again!  Thanks!
